src/systems/physics/Physics.cc

```diff
@@ -1597,6 +1617,13 @@ void PhysicsPrivate::UpdateSim(EntityComponentManager &_ecm) const
         return true;
       });
 
+  // _ecm.Each<components::SlipComplianceCmd>(
```

Comment:
Can the commented block be removed?

Reply:
a1b6d997612965bd16cdd64dabb09585855e1bdf